lib/Varien/Io/File.php

```diff
@@ -377,6 +377,7 @@ public function rmdir($dir, $recursive = false)
      */
     public static function rmdirRecursive($dir, $recursive = true)
     {
+        $result = true;
```

Comment:
Since `unlink` was returning `false` when it can't find the file, shouldn't we switch this to `false`?